README.md

```diff
@@ -125,216 +133,213 @@ print(percl_script.to_json())
+- [PerclCommand](docs/PerclCommand.md)
+- [PerclScript](docs/PerclScript.md)
+- [Play](docs/Play.md)
+- [PlayAllOf](docs/PlayAllOf.md)
+- [PlayBeep](docs/PlayBeep.md)
+- [PlayEarlyMedia](docs/PlayEarlyMedia.md)
+- [PlayEarlyMediaAllOf](docs/PlayEarlyMediaAllOf.md)
+- [QueueList](docs/QueueList.md)
+- [QueueListAllOf](docs/QueueListAllOf.md)
+- [QueueMember](docs/QueueMember.md)
+- [QueueMemberList](docs/QueueMemberList.md)
+- [QueueMemberListAllOf](docs/QueueMemberListAllOf.md)
+- [QueueRequest](docs/QueueRequest.md)
+- [QueueResult](docs/QueueResult.md)
+- [QueueResultAllOf](docs/QueueResultAllOf.md)
+- [QueueResultStatus](docs/QueueResultStatus.md)
+- [RecordUtterance](docs/RecordUtterance.md)
+- [RecordUtteranceAllOf](docs/RecordUtteranceAllOf.md)
+- [RecordUtteranceTermReason](docs/RecordUtteranceTermReason.md)
+- [RecordingList](docs/RecordingList.md)
+- [RecordingListAllOf](docs/RecordingListAllOf.md)
+- [RecordingResult](docs/RecordingResult.md)
+- [RecordingResultAllOf](docs/RecordingResultAllOf.md)
+- [Redirect](docs/Redirect.md)
+- [RedirectAllOf](docs/RedirectAllOf.md)
+- [Reject](docs/Reject.md)
+- [RejectAllOf](docs/RejectAllOf.md)
+- [RemoveFromConference](docs/RemoveFromConference.md)
+- [RemoveFromConferenceAllOf](docs/RemoveFromConferenceAllOf.md)
+- [RequestType](docs/RequestType.md)
+- [Say](docs/Say.md)
+- [SayAllOf](docs/SayAllOf.md)
+- [SendDigits](docs/SendDigits.md)
+- [SendDigitsAllOf](docs/SendDigitsAllOf.md)
+- [SetListen](docs/SetListen.md)
+- [SetListenAllOf](docs/SetListenAllOf.md)
+- [SetTalk](docs/SetTalk.md)
+- [SetTalkAllOf](docs/SetTalkAllOf.md)
+- [Sms](docs/Sms.md)
+- [SmsAllOf](docs/SmsAllOf.md)
+- [StartRecordCall](docs/StartRecordCall.md)
+- [TerminateConference](docs/TerminateConference.md)
+- [TerminateConferenceAllOf](docs/TerminateConferenceAllOf.md)
+- [Unpark](docs/Unpark.md)
+- [UpdateCallRequest](docs/UpdateCallRequest.md)
+- [UpdateCallRequestStatus](docs/UpdateCallRequestStatus.md)
+- [UpdateConferenceParticipantRequest](docs/UpdateConferenceParticipantRequest.md)
+- [UpdateConferenceRequest](docs/UpdateConferenceRequest.md)
+- [UpdateConferenceRequestStatus](docs/UpdateConferenceRequestStatus.md)
 
 ## Documentation For Authorization
 
-
 ## fc
 
 - **Type**: HTTP basic authentication
 
-
 ## Author
 
 support@freeclimb.com
 
-
 ## Notes for Large OpenAPI documents
+
 If the OpenAPI document is large, imports in freeclimb.apis and freeclimb.models may fail with a
 RecursionError indicating the maximum recursion limit has been exceeded. In that case, there are a couple of solutions:
 
 Solution 1:
 Use specific imports for apis and models like:
+
 - `from freeclimb.api.default_api import DefaultApi`
 - `from freeclimb.model.pet import Pet`
 
 Solution 2:
 Before importing the package, adjust the maximum recursion limit as shown below:
+
 ```
 import sys
 sys.setrecursionlimit(1500)
 import freeclimb
 from freeclimb.apis import *
 from freeclimb.models import *
 ```
```

Comment:
General file comment. The only expected change is the `pip install` section. The next time we generate the sdk any manual formatting updates are going to get overridden again and explode the diff. 
Please limit the diff to be only related to the pip install command

Reply:
Updated in latest commit